lib/methods/platform/broadcastStateTransition/broadcastStateTransitionFactory.js

```diff
@@ -23,14 +23,31 @@ function broadcastStateTransitionFactory(grpcTransport) {
     const broadcastStateTransitionRequest = new BroadcastStateTransitionRequest();
     broadcastStateTransitionRequest.setStateTransition(stateTransition);
 
-    const broadcastStateTransitionResponse = await grpcTransport.request(
-      PlatformPromiseClient,
-      'broadcastStateTransition',
-      broadcastStateTransitionRequest,
-      options,
-    );
+    let lastError;
 
-    return BroadcastStateTransitionResponse.createFromProto(broadcastStateTransitionResponse);
+    // TODO: simple retry before the dapi versioning is properly implemented
+    for (let i = 0; i < 3; i += 1) {
+      try {
+        // eslint-disable-next-line no-await-in-loop
+        const broadcastStateTransitionResponse = await grpcTransport.request(
+          PlatformPromiseClient,
+          'broadcastStateTransition',
+          broadcastStateTransitionRequest,
+          options,
+        );
+        return BroadcastStateTransitionResponse.createFromProto(broadcastStateTransitionResponse);
+      } catch (e) {
+        if (e.message !== 'Invalid response: Metadata is not defined') {
```

Comment:
We can check that the error is an instance of `InvalidResponseError`

Reply:
Initially, the fix was about a specific message, but then was replaced with a more broad `instanceof InvalidResponseError`. Resolving the comment